docs/source/cad_resources/autodesk/autodesk.rst

```diff
@@ -21,7 +21,7 @@ Autodesk's desktop-based CAD and Animation tools are world-class and
 available for download. Users must first create an Educational Access
 account, verify your account, and then complete your profile to ensure
 eligibility. Once completed, you'll be provided with one-year education
-access (renewable). To get started, visit the `Autodesk Education Website <https://www.autodesk.com/education/edu-software/overview>`__.
+access (renewable). To get started, visit the `Autodesk Education Website <https://www.autodesk.com/education/home>`__.
```

Comment:
This isn't the same location. The education/home doesn't provide access to software. The issue is that the read timed out - if we double the read timeout, will this solve this problem?